### DISCUSSION TITLE
Plz tell me your favorite board game!!

### DISCUSSION THREAD

**Discussion by disk-12** [NAIST-SE/AttractingContributors#82] [General]
I'm looking for nice board games.
If you've played a board game, please write the title of it!

https://github.com/KACG-D/AttractingContributors

**Comment by teruto725** [comment]
I added poker!

**Comment by manzoku23** [comment]
Added my favorite one!
Please help me
 https://github.com/Exe-dev/AttractingContributors-POKEMON

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames

**Comment by iikyara** [comment]
I added!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Comment by sergiodl-naist** [comment]
Added Risk!
Please visit and try to ask a question or recommend me one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers
